workspace/test_configurations.go

```diff
@@ -209,9 +220,11 @@ var TestConfigurations = map[string]TestConfiguration{
 	"typescript": {
 		Command: "yarn test",
 	},
+	// unison: tests are run from an active UCM session
 	"vbnet": {
 		Command: "dotnet test",
 	},
+	// vimscript: tests are run from inside a vim session
```

Comment:
Looking at how [the test runner does it](https://github.com/exercism/vimscript-test-runner/blob/main/bin/run.sh#L39), it looks like this might be possible. Could you double-check?

Reply:
While this does seem to work, the output is super long and ugly. There doesn't seem to be a `--quiet` flag and I'm disinclined to ship it as is:

```
% vim -XN -i NONE -c ":so hello_world.vim" -c "Vader! hello_world.vader"
Vader note: cannot print to stderr reliably/directly.  Please consider using Vim's -es/-Es option (mode=n).
VIM - Vi IMproved 9.0 (2022 Jun 28, compiled Apr 15 2023 04:26:46)
macOS version - arm64
Included patches: 1-1424
Compiled by root@apple.com
Normal version without GUI.  Features included (+) or not (-):
+acl               -dnd               +listcmds          +postscript        +termresponse
-arabic            -ebcdic            +localmap          +printer           +textobjects
+autocmd           -emacs_tags        -lua               -profile           +textprop
+autochdir         +eval              +menu              -python            +timers
-autoservername    +ex_extra          +mksession         -python3           +title
-balloon_eval      +extra_search      +modify_fname      +quickfix          -toolbar
-balloon_eval_term -farsi             +mouse             +reltime           +user_commands
-browse            +file_in_path      -mouseshape        -rightleft         -vartabs
++builtin_terms    +find_in_path      +mouse_dec         -ruby              +vertsplit
+byte_offset       +float             -mouse_gpm         +scrollbind        +vim9script
+channel           +folding           -mouse_jsbterm     +signs             +viminfo
+cindent           -footer            +mouse_netterm     +smartindent       +virtualedit
-clientserver      +fork()            +mouse_sgr         -sodium            +visual
+clipboard         -gettext           -mouse_sysmouse    -sound             +visualextra
+cmdline_compl     -hangul_input      +mouse_urxvt       +spell             +vreplace
+cmdline_hist      +iconv             +mouse_xterm       +startuptime       +wildignore
+cmdline_info      +insert_expand     +multi_byte        +statusline        +wildmenu
+comments          +ipv6              +multi_lang        -sun_workshop      +windows
+conceal           +job               -mzscheme          +syntax            +writebackup
+cryptv            +jumplist          +netbeans_intg     +tag_binary        -X11
+cscope            -keymap            +num64             -tag_old_static    -xfontset
+cursorbind        +lambda            +packages          -tag_any_white     -xim
+cursorshape       -langmap           +path_extra        -tcl               -xpm
+dialog_con        +libcall           -perl              +termguicolors     -xsmp
+diff              +linebreak         +persistent_undo   +terminal          -xterm_clipboard
+digraphs          +lispindent        +popupwin          +terminfo          -xterm_save
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
  fall-back for $VIM: "/usr/share/vim"
Compilation:
gcc -c -I. -Iproto -DHAVE_CONFIG_H   -DMACOS_X_UNIX  -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
Linking: gcc   -L/usr/local/lib -o vim        -lm -lncurses  -liconv -framework Cocoa

Starting Vader: 1 suite(s), 1 case(s)
  Starting Vader: /Users/david/projects/exercism/vimscript/hello-world/hello_world.vader
    (1/1) [EXECUTE] Say Hi!
  Success/Total: 1/1
Success/Total: 1/1 (assertions: 1/1)
Elapsed time: 0.01 sec.
```